frontend/src/main/components/Courses/BasicCourseTable.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import OurTable from "main/components/OurTable";
+
+export default function BasicCourseTable({ courses }) {
+
+    const columns = [
+        {
+            Header: 'Quarter',
+            accessor: 'quarter',
```

Comment:
Consider displaying this in QYY format instead of YYYYQ format. 

This PR shows how: https://github.com/ucsb-cs156-w22/team04-w22-5pm-courses/pull/52/files

Reply:
Changed format